Documentation/ceph-osd-mgmt.md

```diff
@@ -67,11 +67,11 @@ If you are using `useAllDevices: true`, no change to the CR is necessary.
 
 ### PVC-based cluster
 
-If you want to reduce the storage in your cluster, a healthy PVC-based OSD can be removed as described in this section.
+To reduce the storage in your cluster or remove a failed OSD on a PVC:
 
 1. Shrink the number of OSDs in the `storageClassDeviceSet` in the CephCluster CR.
    - `kubectl -n rook-ceph patch CephCluster rook-ceph --type=json -p '[{"op": "replace", "path": "/spec/storage/storageClassDeviceSets/0/count", "value":<desired number>}]'`
-   - Reduce the `count` of the OSDs to the desired number. Rook will not take any action to automatically remove the extra OSD(s), but will effectively stop managing the orphaned OSD.
+   - Reduce the `count` of the OSDs to the desired number. Rook will not take any action to automatically remove the extra OSD(s).
 2. Identify the orphaned PVC that belongs to the orphaned OSD.
    - The orphaned PVC will have the highest index among the PVCs for the device set.
```

Comment:
This line is not needed anymore since we can remove any OSD having an arbitrary index.

Reply:
And it's necessary to change the following line since PVC names don't have suffix anymore.

> For example if the device set is named set1 and the count was reduced from 3 to 2, the orphaned PVC would have the index 2 and might be named set1-2-data-vbwcf